src/services/postgres_rfqt_firm_quote_validator.ts

```diff
@@ -142,11 +135,65 @@ export class PostgresRfqtFirmQuoteValidator implements RfqtFirmQuoteValidator {
         return takerFillableAmounts;
     }
 
+    private _calculateTakerFillableAmountsFromQuotes(
+        quotes: SignedOrder[],
+        makerLookup: { [key: string]: BigNumber },
+    ): { makerAddressesToAddToCache: string[]; takerFillableAmounts: BigNumber[] } {
+        const makerAddressesToAddToCacheSet: Set<string> = new Set();
+        const takerFillableAmounts = quotes.map(quote => {
+            const makerTokenBalanceForMaker: BigNumber | undefined = makerLookup[quote.makerAddress];
+
+            // TODO: Add Prometheus hooks
+            if (makerTokenBalanceForMaker === undefined) {
+                makerAddressesToAddToCacheSet.add(quote.makerAddress);
+                ORDER_FULLY_FILLABLE.labels(this._workerId);
+                return quote.takerAssetAmount;
+            }
+
+            // Order is fully fillable, because Maker has 100% of the assets
+            if (makerTokenBalanceForMaker.gte(quote.makerAssetAmount)) {
+                ORDER_FULLY_FILLABLE.labels(this._workerId);
+                return quote.takerAssetAmount;
+            }
+
+            // Order is empty, return zero
+            if (quote.makerAssetAmount.lte(0)) {
+                ORDER_NOT_FILLABLE.labels(this._workerId).inc();
```

Comment:
do all the counters need this `inc()` call?

Reply:
yes

Reply:
timers have `set()`, counters have `inc()`

Reply:
it's missing on two of the prometheus calls above ^

Reply:
fixed